Example/Example/SampleContent/video.html

```diff
@@ -4,6 +4,14 @@ <h1>Standard Video</h1>
 <h1>Standard Video no poster image</h1>
 <video src="https://videos.files.wordpress.com/kUJmAcSf/bbb_sunflower_1080p_30fps_normal.mp4" alt="Another video with bunnies"></video>
 
-<h1>WordPress Video Shortcode</h1>
-[video src="https://videos.files.wordpress.com/kUJmAcSf/bbb_sunflower_1080p_30fps_normal.mp4" poster="https://videos.files.wordpress.com/kUJmAcSf/bbb_sunflower_1080p_30fps_normal_scruberthumbnail_2.jpg" alt="Another video with bunnies"/]
+<h1>Standard Video with source element</h1>
+<video alt="Another video with bunnies">
+    <source src="https://videos.files.wordpress.com/kUJmAcSf/bbb_sunflower_1080p_30fps_normal.mp4" type="video/mp4">
+</video>
+
+<h1>Standard Video with multiple source elements</h1>
+<video alt="Another video with bunnies">
+    <source src="https://videos.files.wordpress.com/kUJmAcSf/bbb_sunflower_1080p_30fps_normal.mp4" type="video/mp4">
+    <source src="https://videos.files.wordpress.com/kUJmAcSf/bbb_sunflower_1080p_30fps_normal_fmt1.ogv" type="video/ogg">
```

Comment:
I noticed that in the app this second source is removed:

```html
<h1>Standard Video with multiple source elements</h1>
<p><video alt="Another video with bunnies"><source src="https://videos.files.wordpress.com/kUJmAcSf/bbb_sunflower_1080p_30fps_normal.mp4" type="video/mp4"></video></p>
```

Reply:
Good catch, I updated the code to fix this!